src/views/ID.vue

```diff
@@ -0,0 +1,238 @@
+<template>
+  <div class="home container">
+    <AppHeader />
+    <div class="parent">
+      <div class="searchBox">
+        <input v-show="!popular" type="search" v-model="search" placeholder="Enter acronym" />
+      </div>
+      <div class="buttons">
+        <button @click="showPopular" class="btn buttonSelection">Popular</button>
+        <button @click="showAll" class="btn btn-outline-dark buttonSelection">Show me All</button>
+        <a
+          href="https://github.com/mkfeuhrer/acrons/issues"
+          class="btn buttonSelection"
+          target="blank"
+        >Contribute</a>
+      </div>
+
+      <div class="product-hunt">
+        <a
+          href="https://www.producthunt.com/posts/acrons?utm_source=badge-featured&utm_medium=badge&utm_souce=badge-acrons"
+          target="_blank"
+        >
+          <img
+            src="https://api.producthunt.com/widgets/embed-image/v1/featured.svg?post_id=200817&theme=light"
+            alt="Acrons - One stop solution for finding cool, crazy & popular acronyms | Product Hunt Embed"
+            style="width: 250px; height: 54px;"
+            width="250px"
+            height="54px"
+          />
+        </a>
+      </div>
+
+      <div
+        v-show="!popular"
+        class="searchResult"
+        v-for="(value, index) in filteredList"
+        :key="index"
+      >
+        <div class="grid">
+          <div class="left" :class="{'grayHighlight': index % 2 !== 0 }">{{ value.acronym }}</div>
+          <div
+            class="right"
+            :class="{'grayHighlight': index % 2 !== 0 }"
+          >{{ value.expanded_version}}</div>
+        </div>
+      </div>
+      <div
+        v-show="popular"
+        class="searchResult"
+        v-for="(value, index) in popularList"
+        :key="10000 + index"
+      >
+        <div class="grid">
+          <div class="left" :class="{'grayHighlight': index % 2 !== 0 }">{{ value.acronym }}</div>
+          <div
+            class="right"
+            :class="{'grayHighlight': index % 2 !== 0 }"
+          >{{ value.expanded_version}}</div>
+        </div>
+      </div>
+      <div style="margin-top:20px;">
+        <a href="https://www.buymeacoffee.com/chHAzigTb" target="_blank">
+          <img
+            src="https://cdn.buymeacoffee.com/buttons/default-orange.png"
+            alt="Buy Me A Coffee"
+            style="height: 40px !important;width: 180px !important;"
+          />
+        </a>
+      </div>
+    </div>
+    <AppFooter />
+  </div>
+</template>
+
+<script>
+import json from "../id_techronym_data.json";
+import AppHeader from "@/components/AppHeader.vue";
+import AppFooter from "@/components/AppFooter.vue";
+
+export default {
+  name: "Home",
+  data() {
+    return {
+      search: "",
+      acronymList: json,
+      popular: false,
+      popularList: [
```

Comment:
Can we add 3-4 more to this? Just a single one in popular list doesn't look good.

Reply:
sure

Reply:
done